### PR TITLE
Completed signup page

### PR DESCRIPTION
**Pull request by Sandaru-Dev**
For this pull request, I made the SignUp page completely.

Fixes issue SignUp page #38

SignUp Page

### CODE REVIEW

**Comment by Sandaru-Dev** [comment]
Hi @MadhushaPrasad , This PR is ready for review. Thank you!

File: pages/signup.php [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                <input type="text" name="" id="" placeholder="madhush99">
+
+                <label for="">Password</label>
+                <input type="password" name="" id="" placeholder="********">
+
+                <label for="">Confirm Password</label>
+                <input type="password" name="" id="" placeholder="********">
+            </div>
+
+
```
Comment (supuna97) [inline pages/signup.php:35]: Remove spaces. Follow coding best practices.

Comment (Sandaru-Dev) [reply pages/signup.php:35]: Thank you so much for the comment. I will remove the spaces and will follow the coating practices.

**Review by supuna97 — APPROVED** [review]

File: pages/signup.php [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            </div>
+
+            <div class="position">
+                <input type="radio" name="position" id="">
+                <label for="Student">Student</label>
+                <input type="radio" name="position" id="">
+                <label for="Student">Admin</label>
+            </div>
+            
+            <div class="termscondition">
```
Comment (MadhushaPrasad) [inline pages/signup.php:41]: Can you please change this CSS class name to camelCase or hyphens ?

File: pages/signup.php [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            </div>
+
+            <button type="submit">Create Account</button>
+            <div class="signup">   
+            <hr>
+            <p>Or SIGN up with</p>
+                <hr>
+            </div>
+
+            <div class="socialmedia">
```
Comment (MadhushaPrasad) [inline pages/signup.php:53]: this one also needs to be converted to camelCase or hyphens.

**Review by MadhushaPrasad — CHANGES_REQUESTED** [review]
Mobile responsive is not working properly.

image - (width:425)

image - tablet view (width:768)

So please check responsive for every devices.

File: pages/signup.php (hunk 2 above)
Comment (Sandaru-Dev) [reply pages/signup.php:41]: Thank you so much for your comments, Madhusha. I will revise the code and push it shortly.

File: pages/signup.css [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+.already p{
+    font-size: 16px;
+    color: #000;
+    font-weight: 500;
+    text-align: center;
+}
+
+.position-fixed{
+    position: fixed;
+    /* text-align: center; */
```
Comment (supuna97) [inline pages/signup.css:194]: Remove commented code lines.

File: pages/signup.css [hunk 5]
```diff
@@ -0,0 +1,289 @@
...
+    .form input{
+        width: 550px;
+    }
+
+    .component-1 button{
+        width: 550px;
+    }
+
+    .text-content h4{
+        /* width: 50px; */
```
Comment (supuna97) [inline pages/signup.css:239]: Remove unwanted code lines

File: pages/signup.css [hunk 6]
```diff
@@ -0,0 +1,289 @@
...
+    }
+
+    .component-2 {
+        display: none;
+    }
+
+}
+
+@media screen and (max-width: 468px) {
+
```
Comment (supuna97) [inline pages/signup.css:271]: Remove unwant empty lines.

File: pages/signup.css [hunk 7]
```diff
@@ -0,0 +1,289 @@
...
+    }
+
+    .text-content p{
+        width: 300px;
+    }
+
+}
+
+@media screen and (max-width: 900px) {
+
```
Comment (supuna97) [inline pages/signup.css:251]: Remove unwant empty lines

File: pages/signup.css [hunk 8]
```diff
@@ -0,0 +1,289 @@
...
+    }
+
+    .component-1 button{
+        width: 400px;
+    }
+    
+    .component-2 {
+        display: none;
+    }
+
```
Comment (supuna97) [inline pages/signup.css:288]: Remove unwant empty lines

File: pages/signup.css [hunk 9]
```diff
@@ -0,0 +1,289 @@
...
+    width: 400px;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 18px;
+    text-align: center;
+    color: #1864AB;
+}
+
+@media screen and (max-width: 1068px) {
+
```
Comment (supuna97) [inline pages/signup.css:221]: Remove unwant empty lines

File: pages/signup.css [hunk 10]
```diff
@@ -0,0 +1,289 @@
...
+.social-media{
+    width: 420px;
+    display: flex;
+    flex-direction: column;
+    vertical-align: middle;
+
+}
+.social-media a{
+    /* padding: 8px 30px; */
```
Comment (supuna97) [inline pages/signup.css:163]: Remove commented lines

**Review by supuna97 — CHANGES_REQUESTED** [review]

**Comment by supuna97** [comment]
Hi @Sandaru-Dev  Please check css files and remove unnecessary empty lines. No need for empty lines between css classes.

File: pages/signup.php [hunk 11]
```diff
@@ -0,0 +1,74 @@
...
+    <div class="container">
+        <div class="component-1">
+            <img id="logo" src="../assets/img/logo.svg" alt="logo" width="182px" height="40px" >
+           
+            <h2>Get Started!</h2>
+            <h4>Please sign up to continue</h4>
+
+            <div class="form">
+                <label for="">Email</label>
+                <input type="email" name="" id="" placeholder="mail@example.com">
```
Comment (MadhushaPrasad) [inline pages/signup.php:22]: if you don't use an id for an element then remove this. empty value attributes are not good for security and as well as best practices.

IDs must be unique, but the validator sees id="" as having the same value. So this is bad. please remove those.

File: pages/signup.php [hunk 12]
```diff
@@ -0,0 +1,74 @@
...
+
+                <label for="">User Name</label>
+                <input type="text" name="" id="" placeholder="madhush99">
```
Comment (MadhushaPrasad) [inline pages/signup.php:25]: id issue

File: pages/signup.php [hunk 13]
```diff
@@ -0,0 +1,74 @@
...
+
+                <label for="">Password</label>
+                <input type="password" name="" id="" placeholder="********">
```
Comment (MadhushaPrasad) [inline pages/signup.php:28]: - id issue

- if you use it `for` an attribute then you have to provide that, "for what?" it means you have to tell this for a specific element unique ID. So if you don't want a for attribute then please remove it, if not provide an ID for it.

File: pages/signup.php [hunk 14]
```diff
@@ -0,0 +1,74 @@
...
+
+                <label for="">Confirm Password</label>
+                <input type="password" name="" id="" placeholder="********">
```
Comment (MadhushaPrasad) [inline pages/signup.php:31]: - for issue
- id issue

File: pages/signup.php [hunk 15]
```diff
@@ -0,0 +1,74 @@
...
+
+            <div class="position">
+                <input type="radio" name="position" id="">
+                <label for="Student">Student</label>
+                <input type="radio" name="position" id="">
+                <label for="Student">Admin</label>
+            </div>
+            
+            <div class="terms-condition">
+                <input type="checkbox" name="" id="">
```
Comment (MadhushaPrasad) [inline pages/signup.php:42]: Please provide a value for every `name` attribute that is inside the form tag input elements.
- id issue

File: pages/signup.php [hunk 16]
```diff
@@ -0,0 +1,74 @@
...
+
+            <button type="submit">Create Account</button>
+            <div class="sign-up">   
+            <hr>
+            <p>Or SIGN up with</p>
+                <hr>
+            </div>
+
+            <div class="social-media">
+                <a href="#"> <img src="../assets/img/google-icon.png" alt="">Sign up with Google</a>
```
Comment (MadhushaPrasad) [inline pages/signup.php:54]: Add the value for `alt` which represents the basic idea of the image

File: pages/signup.php [hunk 17]
```diff
@@ -0,0 +1,74 @@
...
+                <a href="#"><img src="../assets/img/facebook-icon.png" alt=""> Sign up with Facebook</a>
```
Comment (MadhushaPrasad) [inline pages/signup.php:55]: Add the value for `alt` which represents the basic idea of the image

**Review by MadhushaPrasad — CHANGES_REQUESTED** [review]

File: pages/signup.css (hunk 10 above)
Comment (Sandaru-Dev) [reply pages/signup.css:163]: Thank you, @supuna97, for your valuable comments. I will make the edits you mentioned. 😊

File: pages/signup.php (hunk 17 above)
Comment (Sandaru-Dev) [reply pages/signup.php:55]: Hi @MadhushaPrasad, Thank you so much for your valuable comments. I will make the modification as you mentioned above. 
Thanks again!

**Review by MadhushaPrasad — CHANGES_REQUESTED** [review]
Original signup page image :
image
- missed some elements and have an additional space bottom of the window
- can't see password elements eye icon both of password and confirm password.
- 
image

tablet view : 
image
- all elemets need to be center.

laptop 1024 view :
image
- little bit deferent than the Figma view

**Comment by Sandaru-Dev** [comment]
Hi @MadhushaPrasad, Thank you so much for your comments. I'm working on addressing those issues and will push the updates as soon as possible. Thank you!

**Comment by MadhushaPrasad** [comment]
> Hi @MadhushaPrasad, Thank you so much for your comments. I'm working on addressing those issues and will push the updates as soon as possible. Thank you!

any update on this PR??

**Comment by Sandaru-Dev** [comment]
Hi @MadhushaPrasad, Thank you so much for your comment. I'm working on the necessary changes and will push them as soon as possible. Thank you!

**Review by supuna97 — APPROVED** [review]

**Comment by Sandaru-Dev** [comment]
Hi @MadhushaPrasad, My PR is waiting for your review. :)

**Comment by MadhushaPrasad** [comment]
> Hi @MadhushaPrasad, My PR is waiting for your review. :)

sure,i will merge this🫡

**Review by MadhushaPrasad — APPROVED** [review]
great job🔥

**Comment by Sandaru-Dev** [comment]
Thank you so much Madhusha. 😊

**Pull request merged**